### PR TITLE
redirect from login.gov -> www.login.gov

### PR DESCRIPTION
**Pull request by jmdembe**
This PR creates a redirect from `login.gov` to `www.login.gov`

### CODE REVIEW

File: templates/_federalist-redirects.njk [hunk 1]
```diff
@@ -445,3 +445,11 @@ server {
   server_name govconnect.18f.gov;
   return 301 https://$target_domain;
 }
+
+# login.gov to www.login.gov
+server {
+  listen {{ PORT }};
+  set $target_domain www.login.gov;
+  server_name login.gov;
```
Comment (amirbey) [inline templates/_federalist-redirects.njk:453]:
```suggestion
  server_name login.gov;
  add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
```

Comment (amirbey) [reply templates/_federalist-redirects.njk:453]: hsts requirement was previously fulfilled for login.gov by fedralist-proxy server
https://github.com/18F/federalist-proxy/pull/108

File: pages.yml [hunk 2]
```diff
@@ -62,3 +62,4 @@
   to: modularcontracting
 - testing-cookbook
 - writing-lab-guide
+- www.login.gov
```
Comment (amirbey) [inline pages.yml:65]:
```suggestion
```

**Review by amirbey — CHANGES_REQUESTED** [review]
hi @jmdembe  ... i've made a few suggestions before merging ... pls review

**Comment by aduth** [comment]
@jmdembe Are we still wanting this? I notice this pull request is draft but there are some recent commits.

**Comment by jmdembe** [comment]
> @jmdembe Are we still wanting this? I notice this pull request is draft but there are some commits more recent than the last round of review.

This can be closed. This was resolved between the Cloud.gov team and DevOps instead of through this ticket.

**Pull request closed without merging**